src/packages/serializer/index.js

```diff
@@ -386,7 +389,7 @@ class Serializer {
         [...this.hasOne, ...this.hasMany].reduce((hash, name) => ({
           ...hash,
 
-          [name]: (async () => {
+          [dasherize(underscore(name))]: (async () => {
```

Comment:
`camelCase` relationship names now must be normalized before their added as keys to a [JSONAPI Relationships Object](http://jsonapi.org/format/#document-resource-object-relationships).